fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1033,20 +1035,30 @@ public void rebuildIndex(@Nonnull FDBRecordStore store) {
     @VisibleForTesting
     @Nonnull
     CompletableFuture<Void> buildIndexAsync(boolean markReadable) {
+        KeyValueLogMessage message = KeyValueLogMessage.build("online build index",
```

Comment:
This is minor, and if you like `"online build index"`, that's fine, but I think `"online index build"` or `"build index online"` sound better to me. But this is also fine.

Reply:
Changed to "build index online"